spec/controllers/api/jobs_controller_spec.rb

```diff
@@ -3,25 +3,83 @@
 require './spec/rails_helper'
 
 RSpec.describe Api::JobsController, type: :controller do
+  before do
+    ENV['JOB_BOARD_PASSWORD'] = 'wnb.rbRULEZ!'
+    ENV['JWT_HMAC_SECRET'] = '1234567'
+  end
+
+  after do
+    ENV['JOB_BOARD_PASSWORD'] = nil
+    ENV['JWT_HMAC_SECRET'] = nil
+  end
+
   describe 'GET #index' do
```

Comment:
I've added more tests to this endpoint:
* token is expired
* token is invalid (encrypted with a different secret key)
* token is valid